Code/Engine/RendererCore/Lights/Implementation/LightComponent.cpp

```diff
@@ -19,12 +19,15 @@ void ezLightRenderData::FillBatchIdAndSortingKey(float fScreenSpaceSize)
 //////////////////////////////////////////////////////////////////////////
 
 // clang-format off
-EZ_BEGIN_ABSTRACT_COMPONENT_TYPE(ezLightComponent, 4)
+EZ_BEGIN_ABSTRACT_COMPONENT_TYPE(ezLightComponent, 5)
 {
   EZ_BEGIN_PROPERTIES
   {
-    EZ_ACCESSOR_PROPERTY("LightColor", GetLightColor, SetLightColor),
+    EZ_ACCESSOR_PROPERTY("UseColorTemperature", GetUsingColorTemperature, SetUsingColorTemperature),
+    EZ_ACCESSOR_PROPERTY("LightColor", GetBaseLightColor, SetLightColor),
```

Comment:
Btw this should GetLightColor and not GetBaseLightColor, then it will also work with the icon coloring when using color temperature.